climakitae/util/utils.py

```diff
@@ -1183,3 +1185,82 @@ def stack_sims_across_locs(ds):
         for sim_name in ds["simulation"]
     ]
     return ds
+
+
+def clip_to_shapefile(
+    data: xr.Dataset | xr.DataArray,
+    shapefile_path: str,
+    feature: tuple[str, str | int | float | bool | list[str | int | float | bool]] = (),
```

Comment:
In this unique circumstance I would list this as `tuple[str, Any]` and `from typing import Any` since it looks like you're not really restricting the second value, but the first value MUST be string